hostsensorutils/hostsensorgetfrompod.go

```diff
@@ -104,7 +105,41 @@ func (hsh *HostSensorHandler) GetLinuxSecurityHardeningStatus() ([]hostsensor.Ho
 // return list of KubeletCommandLine
 func (hsh *HostSensorHandler) GetKubeletCommandLine() ([]hostsensor.HostSensorDataEnvelope, error) {
 	// loop over pods and port-forward it to each of them
-	return hsh.sendAllPodsHTTPGETRequest("/kubeletCommandLine", "KubeletCommandLine")
+	resps, err := hsh.sendAllPodsHTTPGETRequest("/kubeletCommandLine", "KubeletCommandLine")
+	if err != nil {
+		return resps, err
+	}
+	for resp := range resps {
+		data := makeCmdLineMap(string(resps[resp].Data))
+		resBytesMarshal, err := json.Marshal(data)
+		// TODO catch error
+		if err == nil {
+			resps[resp].Data = json.RawMessage(resBytesMarshal)
+		}
+	}
+
+	return resps, nil
+
+}
+
+func makeCmdLineMap(fullCommand string) map[string]interface{} {
+
+	commands := strings.Split(fullCommand, " ")
+	if len(commands) < 1 {
+		return nil
```

Comment:
@Daniel-GrunbergerCA  Because you do not handle nil return, you should return here an empty map and not nil